diff --git client_generic.orig/ContentDecoder/graph_playlist.h client_generic/ContentDecoder/graph_playlist.h
index 1f0a6cd..5d9b8d7 100644
--- client_generic.orig/ContentDecoder/graph_playlist.h
+++ client_generic/ContentDecoder/graph_playlist.h
@@ -13,13 +13,12 @@
 
 #include	"boost/filesystem/path.hpp"
 #include	"boost/filesystem/operations.hpp"
-#include	"boost/filesystem/convenience.hpp"
+#include	"boost/filesystem.hpp"
 
 using boost::filesystem::path;
 using boost::filesystem::exists;
 using boost::filesystem::no_check;
 using boost::filesystem::directory_iterator;
-using boost::filesystem::extension;
 
 namespace ContentDecoder
 {
